tests/test_servo.py

```diff
@@ -307,30 +315,59 @@ def test_load_configuration_to_subnode_zero(read_config, pytestconfig, connect_t
 
 @pytest.mark.canopen
 @pytest.mark.ethernet
-def test_store_parameters(connect_to_slave):
+@pytest.mark.ethercat
+def test_store_parameters(connect_to_slave, request):
+    user_over_voltage_register = "DRV_PROT_USER_OVER_VOLT"
+
     servo, net = connect_to_slave
-    assert servo is not None and net is not None
+
+    initial_user_over_voltage_value = servo.read(user_over_voltage_register)
+    new_user_over_voltage_value = initial_user_over_voltage_value + 5
+
+    servo.write(user_over_voltage_register, new_user_over_voltage_value)
+
+    assert servo.read(user_over_voltage_register) == new_user_over_voltage_value
 
     servo.store_parameters()
 
-    value = servo.read("DRV_STATE_STATUS")
-    assert value is not None
+    time.sleep(5)
```

Comment:
No way of knowing if the store_parameters has been completed right....?

Reply:
I tried it but it did not work for all the drives.

Reply:
Not if it fails.
Maybe do a 
```
try:
    modify_value()
finally:
    restore_value()
```

or on the fixture just do a load config at the beginning of each test.